packages/typespec-client-generator-core/test/rules/require-client-suffix.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BasicTestRunner,
+  createLinterRuleTester,
+  LinterRuleTester,
+} from "@typespec/compiler/testing";
+import { beforeEach, describe, it } from "vitest";
+import { requireClientSuffixRule } from "../../src/rules/require-client-suffix.js";
+import { createTcgcTestRunner } from "../test-host.js";
+
+describe("typespec-client-generator-core: client names end with 'Client'", () => {
```

Comment:
you can remove the top level describe now, with vitgest it adds extra nesting otherwise